src/VisualStudio/CSharp/Impl/Options/IntelliSenseOptionPageControl.xaml.cs

```diff
@@ -39,5 +43,19 @@ private void BringUpOnIdentifier_Unchecked(object sender, System.Windows.RoutedE
             ShowKeywords.IsEnabled = true;
             ShowSnippets.IsEnabled = true;
         }
+
+        private void Show_completion_list_after_a_character_is_deleted_Checked(object sender, RoutedEventArgs e)
```

Comment:
THe issue here is that this a ```bool?``` and not a ```bool```.  BindToOption can't bind check boxes to nullable bools.